GEOSwift/GEOSwift
# Carthage

Firstly for people who just want it to work I have uploaded a binary that Carthage can use:

Cartfile
```
github "madhavajay/GEOSwift" ~> 3.2.0
```

I also managed to almost get this to work as part of the original project.
To get Carthage to build this as a dependency I had to do the following:
- Removing Test Target during Framework build
- Test target requires code signing
- Remove missing xcconfigs which stop build, pod install fixes anyway
- Added Carthage path to Check Pods Manifest script
- If called from Carthage, simply run pod install to get dependency

This works except that the sub dependency geos inside the cocopod is not built yet.
I am pretty sure that can be fixed a few ways:
- Give the geos project its own xcodeproject file which builds the framework and add it as a Cartfile dependency to the main GEOSwift project
- Add a separate fork of geos with Carthage
- Find a way to trigger the geos build with cocoapods from the command line and add it to the CARTHAGE branch of the Check Pods Manifest script

Also @macdrevx another solution and probably a good idea anyway is for you to simply upload a zip file of the iOS framework when it's finished building to the release page. Carthage automatically grabs these if they exist.
